.github/workflows/test.yaml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+      - staging
```

Comment:
Should we call the branch `staging` or just `dev` like we do for our other repos?

Reply:
Maybe :shrug:

Reply:
I prefer `dev`.